arcgis-java-toolkit/README.md

```diff
@@ -0,0 +1,108 @@
+# arcgis-java-toolkit
+
+## Introduction
+
+The ArcGIS Runtime SDK for Java Toolkit contains controls and utilities to simplify your app development. The toolkit is provided as an open source resource, so you can feel free to download or clone the code and customize to meet your requirements.
```

Comment:
API was used above, could update here for consistency? If so there are other instances in this readme to update

```suggestion
The ArcGIS Runtime API for Java Toolkit contains controls and utilities to simplify your app development. The toolkit is provided as an open source resource, so you can feel free to download or clone the code and customize to meet your requirements.
```